src/Sylius/Bundle/ApiBundle/Serializer/CommandOperationDenormalizer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use Symfony\Component\Serializer\Exception\MissingConstructorArgumentsException;
+use Symfony\Component\Serializer\Normalizer\ContextAwareDenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+final class CommandOperationDenormalizer implements ContextAwareDenormalizerInterface
```

Comment:
```suggestion
final class CommandConstructorDenormalizer implements ContextAwareDenormalizerInterface
```
as we are checking the arguments of a constructor, but I'm not convinced by any of these names

Reply:
What about just `CommandDenormalizer/Normalizer`? We have an **Address**Denormalizer or **ProductVariant**Normalizer, which indicates the resource on which we operate, but these are more generic services, that's why I wanted to call them a little bit differently